androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/TripStops.kt

```diff
@@ -238,7 +248,7 @@ fun TripStops(
             }
             StopList(
                 splitStops.followingStops,
-                lastStopSequence?.plus(if (splitStops.isTruncatedByLastAlert) 1 else 0),
+                lastStopSequence,
```

Comment:
During an earlier revision of #900 I was truncating the stops when the TripDetailsStopList was first being constructed, and when I was doing that this was necessary to distinguish a truncated stop from an actual last stop, but since I’m truncating in splitForTarget instead, this isn’t needed.